doc/source/release-notes/Percona-Server-5.7.10-1.rst

```diff
@@ -47,4 +47,6 @@ Bugs Fixed
 
  Removed unnecessary code in InnoDB error monitor thread. Bug fixed :bug:`1521564` (upstream :mysqlbug:`79477`).
 
+ After completed ``ALTER TABLE`` queries on the temporary table would produce warnings on indexes that were not found in the index translation table. Bug fixed :bug:`1233431`.
```

Comment:
With /feature link to expanded fast index creation/ enabled, DDL queries involving InnoDB temporary tables would cause later queries on the same tables to produce warnings that their indexes were not found in the index translation table.